lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -3578,4 +3578,22 @@ public function getStringLiteralQuoteCharacter()
     {
         return "'";
     }
+
+    /**
+     * Escapes metacharacters in a string intended to be used with a LIKE
+     * operator.
+     */
+    public function escapeStringForLike(string $untrustedString): string
+    {
+        $replacePairs = [];
```

Comment:
The escape character itself should also be always escaped. Does this approach account for `getLikeMetaCharacters()` always returning `getLikeEscapeChar()` or it `escapeStringForLike()` should handle that itself?

Reply:
I think the best is to let `escapeStringForLike` handle it, and I could rename "meta" to "wildcard", what do you think?

Reply:
Agree on both.